src/odemis/util/conversion.py

```diff
@@ -164,6 +164,33 @@ def hex_to_frgba(hex_str, af=1.0):
     """
     return rgba_to_frgba(hex_to_rgba(hex_str, int(af * 255)))
 
+# from claude / ome_types
+def rgba_to_signed_int32(rgb, a: int = 255):
```

Comment:
Let's name this function `rgba_to_int32`, so that the name is symmetrical to the opposite function. Typically "int32" means "signed int", so no need to make the function name extra long.